conf/distro/openxt-main.conf

```diff
@@ -73,9 +73,13 @@ PREFERRED_PROVIDER_virtual/java-native = "jamvm-native"
 PREFERRED_PROVIDER_virtual/javac-native = "ecj-bootstrap-native"
 PREFERRED_PROVIDER_jpeg-native = "jpeg-native"
 
-# Set the preferred version to c-xenstored.
-# xen-xenstored-c is a PROVIDE alias to xen.bb.
-PREFERRED_PROVIDER_xen-xenstored = "xen-xenstored-c"
+# Set the preferred version to C xenstored from xen-tools
+PREFERRED_PROVIDER_virtual/xenstored = "xen-tools"
+PREFERRED_RPROVIDER_virtual/xenstored = "xen-tools-xenstored"
+
+# The C and OCaml xenstored packages can be installed together
+# and this whitelisting is necessary to allow both to be built
+MULTI_PROVIDER_WHITELIST += " virtual/xenstored"
```

Comment:
Why do we need to build both simultaneously?

Reply:
We need both the `xen-tools` and `xen-ocaml-libs` recipes to be built and the two xenstored imlementations are built by those recipes. Without whitelisting this, the build will fail since it believes that the two packages collide.

Reply:
Ack